content/spin/install.md

```diff
@@ -22,7 +22,19 @@ Spin runs on Linux (amd64 and arm64), macOS (Intel and Apple Silicon), and Windo
 
 {{ startTab "Linux"}}
 
-There are multiple ways to install Spin. The easiest is to use the installer script, hosted on this site. The script installs Spin along with a starter set of language templates and plugins:
+You can manage your Spin installation via [Homebrew](https://brew.sh/). Homebrew automatically installs Spin templates and Spin plugins, and on uninstall, will prompt you to delete the directory where the templates and plugins were downloaded:
+
+Add Spin to the list of formulae that Homebrew tracks, updates, and installs from:
```

Comment:
technically that is not what this command does. Rather it fetches Fermyon's homebrew tap
```suggestion
Install the Fermyon tap, which Homebrew tracks, updates, and installs Spin from:
```